### PR TITLE
Added Support for Django20,21,22,3 and python 3.6 and 3.7

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-Added python 3.6 and 3.7 support …
-Added Django20,21,22,3 Support
-Updated tox and travis config files
-Updated documentation
-Updated Authors
-Version bump, Major release

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,34 @@
 sudo: false
```
Comment (Ayub-Khan) [inline .travis.yml:1]: Updated travis file according to tox file.

File: HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,14 @@
 History
 -------
 
+0.6.0 (2020-01-07)
```
Comment (Ayub-Khan) [inline HISTORY.rst:6]: Major release as it adds support for a lot of new versions.

File: setup.py [hunk 3]
```diff
@@ -35,9 +35,9 @@ def read(*parts):
 
 
 install_requires = [
-    'django>=1.8,<2.0',
+    'django>=1.11,<3',
```
Comment (Ayub-Khan) [inline setup.py:38]: Remove support for django<1.11 as those versions are too old and not used much. Projects using django<1.11 can add version constraint until they have upgraded their django version.

File: setup.py [hunk 4]
```diff
@@ -35,9 +35,9 @@ def read(*parts):
...
     'babel>=1.3',
-    'django-babel>=0.5.1',
+    'enmerkar==0.7.0',
```
Comment (Ayub-Khan) [inline setup.py:40]: Removed django-babel and used enmerkar as it is well maintained as compared to django-babel. Hard coding version as newer version drops support for django<2.1.

File: setup.py [hunk 5]
```diff
@@ -77,11 +77,21 @@ def read(*parts):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Natural Language :: English',
+        'Programming Language :: Python',
```
Comment (Ayub-Khan) [inline setup.py:80]: some enhancements.

File: src/django_babel_underscore/__init__.py [hunk 6]
```diff
@@ -1,13 +1,18 @@
 # -*- coding: utf-8 -*-
 import django
 
-if django.VERSION[:2] >= (1, 8):
-    from django.template.base import Lexer, TOKEN_TEXT
-else:
-    from django.template import Lexer, TOKEN_TEXT
+import django
```
Comment (Ayub-Khan) [inline src/django_babel_underscore/__init__.py:4]: Updated to support newer versions of Django.

File: tox.ini [hunk 7]
```diff
@@ -1,95 +1,21 @@
+[tox]
```
Comment (Ayub-Khan) [inline tox.ini:1]: Updated the tox file to fix the issue where travis was not able to install the correct version of django and to simplify things.

File: tox.ini [hunk 8]
```diff
@@ -1,95 +1,21 @@
...
-deps = {[testenv]deps19}
-
-[testenv:pypy-1.10.x]
-basepython = pypy
-deps = {[testenv]deps110}
-
-[testenv:pypy-1.11.x]
-basepython = pypy
-deps = {[testenv]deps111}
+    python -Wd -m pytest {posargs}
```
Comment (Ayub-Khan) [inline tox.ini:16]: To be able to see deprecation warnings.

**Comment by Ayub-Khan** [comment]
@EnTeQuAk Kindly review.

**Comment by jmbowman** [comment]
@EnTeQuAk Do you think you'll have time to review this sometime in the near future?  This package is used in Open edX, which we're currently updating to use Django 2.2 instead of 1.11.  Django 1.11 support doesn't end until April, but we'd like to get all of our services updated as soon as possible so that other institutions running Open edX have more time to upgrade before the support cutoff.

**Pull request closed without merging**